development/client/src/App.js

```diff
@@ -1,37 +1,109 @@
-import React from "react";
-import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
-import { useQuery } from "@apollo/client";
+import React, { useState } from "react";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import {
+  ApolloClient,
+  ApolloProvider,
+  InMemoryCache,
+  makeReference,
+  useQuery,
+} from "@apollo/client";
 import ColorSchemeGenerator from "./ColorSchemeGenerator";
 import Favorites from "./Favorites";
 import ColorLookup from "./ColorLookup";
 import { GET_SAVED_COLORS } from "./queries";
 import "./App.css";
 
 function App() {
```

Comment:
To make this testable, I had to move some code around so that I could recreate the client and ensure it was set in context correctly.